src/encoded/static/components/objectutils.js

```diff
@@ -618,11 +618,24 @@ export class ImageWithFallback extends React.Component {
         });
     }
 
-    // Display default "not found" image for non-existent image src
     onError() {
+        let imageAlt = null;
+        let imageUrl = null;
+
+        // IE11 has an issue where it frequently throws a "Permission denied" exception, when the image
+        // exist. This workaround either shows the image or if not found shows the browser's
+        // inbuilt no-image display
+        if (document && document.all !== undefined) {
+            imageUrl = this.state.imageUrl;
+            imageAlt = this.state.imageAlt;
+        } else {
+            imageUrl = '/static/img/brokenImage.png';
+            imageAlt = 'Not found';
+        }
+
         this.setState({
-            imageUrl: '/static/img/brokenImage.png',
-            imageAlt: 'Not found',
+            imageUrl,
+            imageAlt,
```

Comment:
This is a pretty weird bug.

Just wondering if we need the setState outside the if/else. Maybe it could just be in the `else` because for the `if` case, imageAlt and imageUrl are already saved in state.

Reply:
@emro Code updated.